sbin/newfs_msdos/newfs_msdos.8

```diff
@@ -159,6 +160,9 @@ Number of hidden sectors.
 Number of reserved sectors.
 .It Fl s Ar total
 File system size.
+.It Fl T Ar timestamp
```

Comment:
I notice NetBSD has `At timestamp`; I guess it's just a typo. mandoc renders it as `-T AT&T UNIX timestamp` :)